src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -536,8 +536,9 @@ TEST_CASE("Concurrency - shared memory infer input calls")
 
   tcm.infer_data_manager_ =
       MockInferDataManagerFactory::CreateMockInferDataManager(
-          params.batch_size, params.shared_memory_type, params.output_shm_size,
-          mip.mock_model_parser_, tcm.factory_, mip.mock_data_loader_);
+          params.max_threads, params.batch_size, params.shared_memory_type,
```

Comment:
Is the existing set of testing good enough to cover the refactor/design changes in this PR?

Reply:
The existing tests were not quite enough. There are a few new tests in this PR